### DISCUSSION TITLE
One feature not added.

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#4992] [Bugs]
Collected with Trimble phone.

I collected 5 points in the field, 4 of them where pushed and one when raised an error:

14:08:47.522 QGSMSGLOG WARNING  Commit errors:
  ERROR: 1 feature(s) not added - geometry type is not compatible with the current layer.
14:08:47.523 qfieldcloud.qgis.apply_deltas WARNING  Error while applying a single delta: Failed to commit changes
14:08:47.530 QGSMSGLOG WARNING  Unbalanced call to leaveUpdateMode() w.r.t. enterUpdateMode()
14:08:47.723 qfieldcloud.qgis.apply_deltas INFO     Some deltas have not been applied

**Comment by suricactus** [comment]
Can you please provide a copy of your project and QField version you have tested with?
